samples/ControlCatalog.NetCore/Program.cs

```diff
@@ -124,6 +125,7 @@ public static AppBuilder BuildAvaloniaApp()
                     EnableIme = true
                 })
                 .UseSkia()
+                .WithFonts(new InterFontCollection())
```

Comment:
In the future, how would user define async font collection? If font collection is added to the manager after text is rendered (let's say, user has loaded a font and added a font collection there). Will text be re-rendered, like in the html is sometimes is (when fonts are slowly loaded)? Or is it not supported yet?

Reply:
That isn't supported yet. Font collections must be added to the font manager before use. We could introduce a markup extension that does lazy loading in the background.